pymc_marketing/mmm/hsgp.py

```diff
@@ -260,6 +260,19 @@ class HSGPBase(BaseModel):
     )
     X_mid: float | None = Field(None, description="The mean of the training data")
     dims: Dims = Field(..., description="The dimensions of the variable")
+    transform: str | None = Field(
+        None,
+        description="Optional transformation for the variable",
```

Comment:
Can you give more details on the type of transformations? Maybe giving examples like "exp" and "sigmoid" and mentioning they have to by in the pytensor.tensor namespace (or similar)

Reply:
Done. Thanks for the suggestion

Reply:
Great thanks!